.github/actions/post-deploy-smoke-test/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Smoke test post deploy
+description: Invoke a script that visits a deploy smoke check page that displays whether the backend / db are healthy.
+inputs:
+  deploy-env:
+    description: The environment being deployed (e.g. "prod" or "test")
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: create env file
+      shell: bash
+      working-directory: frontend
+      run: |
+        touch .env
+        echo REACT_APP_BASE_URL=https://${{ inputs.deploy-env }}.simplereport.gov >> .env.production.local
```

Comment:
I think if we are setting this here this wouldn't catch the issue that we had from this outage: https://github.com/cdcent/simplereport_docs/blob/main/incident_postmortem/2023-10-04_frontend_unable_to_communicate_with_backend.md because our frontend was using the value set by the `base_domain_name` https://github.com/CDCgov/prime-simplereport/pull/7057/files#diff-3238e23f805344543ab5d80091a03bc6d514825c2d8ab893e48526d61d6f6de7L43 -- Let me know if I'm understanding this correctly 😅

Reply:
Ahh good catch. I think we might need to pull that environment variable up into GitHub so that we can share it between workflows then.

Reply:
Updated this to use the prod-env variable! Lmk if you have any other concerns @emyl3